nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java

```diff
@@ -118,17 +120,25 @@ public void testMetadataProcessing() throws Exception
     final String agricolae131Content =
         new String(Files.readAllBytes(testData.resolveFile(PACKAGES_AGRICOLAE_131_NAME).toPath()));
 
-    //Verify PACKAGES(metadata) contain appropriate content about R package.
+    // Wait till metadata is created
+    Thread.sleep(1000);
```

Comment:
Same as the above comment, probably constant should be static for reusing from here

Reply:
done in fe40919080a00ea9162d0af9be02c414af338e27